### ISSUE TITLE
AUGraph Deprecation. Future for StreamingKit?

### ISSUE THREAD

**Issue by xcourier** [tumtumtum/StreamingKit#443]
Since Apple is formally [deprecating AudioGraph](url) in favor of AVAudioEngine going forward, are there any plans to migrate this great project towards Swift and the newer AVAudioEngine Framework? 

Perhaps there is already something else that can replace it? 

https://developer.apple.com/documentation/ios_ipados_release_notes/ios_ipados_13_beta_7_release_notes?fbclid=IwAR3GrEym2tTS5cD-OifDW0KojZKZVCCj8z99Ti9n3Xo8oV-9b71Pekjaz7o

**Comment by iDevelopper** [comment]
Yes, It would be cool to fix all the deprecated methods and warnings! But why Swift? Objective-c is very stable and can be used easily with a Swift project...

**Comment by xcourier** [comment]
I'd be fine with Obj-C -- It's pure conjecture, but I figured if someone took on porting StreamingKit from AUGraph to AVAudioEngine it would in essence be 'starting over' to some degree...And if that were the case it may be better to do it in Swift to future-proof it.

**Comment by dimitris-c** [comment]
Hey, just release my version of StreamingKit in Swift with AVAudioEngine, not battle tested but worth a try. Will add a readme and coocapods/spm/carthage support in the coming days.
**https://github.com/dimitris-c/AudioStreaming**

All thanks should go to @tumtumtum 🙇‍♂️ for StreamingKit

**Comment by CaseyIT** [comment]
Hey @dimitris-c . Thanks for taking this on. Looks promising!

**Comment by xcourier** [comment]
@dimitris-c Looks promising! Look forward to trying it out.
